.github/workflows/run_tests.yaml

```diff
@@ -29,8 +29,9 @@ jobs:
         python -m pip install --upgrade pip
         pip install flake8 pytest
         export GSLDIR=$(gsl-config --prefix)
-        if [ -f NuRadioMC/test/requirements.txt ]; then pip install -r NuRadioMC/test/requirements.txt; fi
-        export PYTHONPATH=$PWD:$PYTHONPATH
+        python install_dev.py --install --dev proposal --no-interactive
+        pip uninstall numba -y # easiest way to test ARZ without numba
```

Comment:
I assume this is required because numba is a requirement in the install_dev.py script?

Reply:
Yes - we should probably decide whether `numba` and `uproot` should be core dependencies or optional.
If optional, we keep NuRadioMC 'smaller', but a pip install will not install optional dependencies. A missing `numba` install will not result in errors, but just in a slower ARZ signal generation, which 'casual' users might never catch. Uproot on the other hand is only required for specific experiments (as far as I understand?) and will generate a simple `ImportError` if the user tries to use modules that depend on it. Hence why `uproot` is currently optional, and `numba` is core. But I am happy to change this.

Reply:
to me that looks good, just wanted to make sure I understand it.

Reply:
Both uproot and numba are now core dependencies again. Christian's comment below for completeness:
>I think numba is fine, uproot too. These are well maintained packages. And we always want numba because of the speed gain